drag_and_drop_v2/public/js/drag_and_drop_edit.js

```diff
@@ -465,11 +458,14 @@ function DragAndDropEditBlock(runtime, element, params) {
                                 name = $el.find('.item-text').val(),
                                 imageURL = $el.find('.item-image-url').val(),
                                 imageDescription = $el.find('.item-image-description').val();
+                                checkedCheckboxes = $el.find('.zone-checkbox:checked')
```

Comment:
@haikuginger I think it would be worth renaming this to `selectedZones`, to focus more on the information that this code is supposed to retrieve (and focus less on the current UI choice of using checkboxes for zone selection, which is arbitrary).

Also, you'll need to add `;` at the end of this line and replace `;` in the previous line with `,`.